package.json

```diff
@@ -53,9 +55,12 @@
     "remark-html": "^13.0.1",
     "remark-rehype": "^8.0.0",
     "rss": "^1.2.2",
+    "stream": "^0.0.2",
```

Comment:
gotta remove some of these that I added in for testing out different libraries.

Reply:
We might want to move these action-related dependencies to `devDependencies` so that our bundle doesn't get bloated with dev-related stuff.

Reply:
good idea! I'll move those other to `devDependencies` 😄